src/main/java/me/jellysquid/mods/sodium/client/gui/SodiumOptionsGUI.java

```diff
@@ -197,7 +198,7 @@ private void renderOptionTooltip(MatrixStack matrixStack, ControlElement<?> elem
         OptionImpact impact = option.getImpact();
 
         if (impact != null) {
-            tooltip.add(new LiteralText(Formatting.GRAY + "Performance Impact: " + impact.toDisplayString()));
+            tooltip.add(new TranslatableText("options.tooltips.performance_impact_string", Formatting.GRAY, impact.toDisplayString()));
```

Comment:
Might make sense to not include the formatting in the translation string itself, and rather set the style on the `Text` directly

Reply:
Just tried it, turns out that that breaks the special formatting on the `impact.toDisplayString()` call so I'm not going to include that here.